packages/core/src/baseclient.ts

```diff
@@ -565,7 +565,7 @@ export abstract class BaseClient<O extends ClientOptions> implements Client<O> {
 
     if (this._isEnabled() && this._transport) {
       return this._transport.send(envelope).then(null, reason => {
-        DEBUG_BUILD && logger.error('Error while sending event:', reason);
+        DEBUG_BUILD && logger.error('Error while sending envelope:', reason);
```

Comment:
Given that `sendEnvelope` is called for multiple envelope item types (not just events), I opted to slightly change the message here to reflect this